config/locales/two_factor_authentication/en.yml

```diff
@@ -1,7 +1,22 @@
 ---
 en:
   two_factor_authentication:
+    account_reset:
```

Comment:
Are we confident that none of these are used by the two_factor_authentication gem? I was under the impression that was why we had these under the "devise" key.

Reply:
Yep. None of these are triggered by our app because we basically don't use any of the gem's controller: https://github.com/Houdini/two_factor_authentication/blob/master/config/locales/en.yml

Reply:
Cool. I wonder at what point we should just extract the gem since with the multiple phone configurations presumably we won't be using the model code either?

Reply:
Not suggesting we do that here, of course. Just wondering.

Reply:
Yeah, that will probably need to happen at some point. It would be great if we wrote our own gem.